index.js

```diff
@@ -84,6 +84,192 @@ app.post('/add/:quizName/:count', urlencodedParser, async (req, res) => {
     </html>`)    
 })
 
+////
+////
+////These are the /take routes
+
+app.post('/take', urlencodedParser, async(req, res) => {
+    console.log("This makes sure that this is running properly")  // TEMP
+    
+    var quizName = req.body.userName
+    const inCase = 'Sorry, this quiz does not exist.'
+    if (quizName == '') {
```

Comment:
Good thought on validating the user input, but empty string is just as invalid as an incorrect quiz name. Therefore you probably want to do something different if the quiz is not found or user name is empty: immediately return an error page, instead of the normal success page.

Reply:
For the sake of clarity, the quizName should remain the quiz name and not become something else like an error message.